src/components/exercises/HasPhoneme.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import { View } from 'react-native';
+
+import speakerImage from '../../assets/images/speaker.png';
+
+import { PRIMARY } from '../../styles/colors';
+import { RoundTextButton, RoundImageWithButton } from '../Components';
+import { loadSounds } from '../helpers/audio';
+
+class HasPhoneme extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      highlighted: null
+    };
+  }
+
+  play = () => {
+    const playAll = ([sound, ...rest]) => {
```

Comment:
Since you only have one single sound:
```.js
  const { sound } = this.props;

  const play = () => {
    sound.playAsync();
    sound.setPlaybackFinishedCallback(() => {
      sound.setPositionAsync(0);
    });
  };
```